ci/conda_env_python.txt

```diff
@@ -17,7 +17,7 @@
 
 # don't add pandas here, because it is not a mandatory test dependency
 boto3  # not a direct dependency of s3fs, but needed for our s3fs fixture
-cffi
+cffi<1.16.0  # cffi 1.16.0 with Python 3.9 has an incompatibility with grpcio (required for GCS testbench)
```

Comment:
Good catch!

Can we restrict this pin only for Python 3.9? It seems that we can do it with `requirements.txt` but I don't know whether we can do it with conda...

Reply:
I don't know if it's possible with conda. The only docs I could find about the version specification syntax was this https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf